lightly/utils/benchmarking/linear_classifier.py

```diff
@@ -153,9 +156,77 @@ def configure_optimizers(  # type: ignore[override]
             ),
             "interval": "step",
         }
+
         return [optimizer], [scheduler]
 
+
+class LinearClassifier(BaseClassifier):
+    def __init__(
+        self,
+        model: Module,
+        batch_size_per_device: int,
+        lr: float = 0.1,
+        feature_dim: int = 2048,
+        num_classes: int = 1000,
+        topk: Tuple[int, ...] = (1, 5),
+    ) -> None:
+        super().__init__(
+            model,
+            batch_size_per_device,
+            lr,
+            feature_dim,
+            num_classes,
+            topk,
+        )
+
+    def forward(self, images: Tensor) -> Tensor:
+        # For linear evaluation, we want to freeze the feature extractor.
+        with torch.no_grad():
+            features = self.model.forward(images).flatten(start_dim=1)
+
+        output: Tensor = self.classification_head(features)
+
+        return output
+
+    def get_trainable_parameters(self) -> List[torch.nn.Parameter]:
```

Comment:
```suggestion
    def get_trainable_parameters(self) -> List[Parameter]:
```

We always import classes directly: 
```
from package import Class
```
 and import modules for functions
```
import torch
t = torch.tensor([1, 2, 3])
```